src/main/java/org/cactoos/scalar/Reduced.java

```diff
@@ -97,6 +97,21 @@ public Reduced(
         this.function = reduce;
     }
 
+    /**
+     * Ctor.
+     * @param values The scalars
+     * @param reduce Reducing function
+     */
+    public Reduced(
```

Comment:
@andreoss if you introduce a new secondary constructor, put it before the primary, not after